intersection.py

```diff
@@ -0,0 +1,43 @@
+#As taught iin class using binary search to solve this problem
+#Time Complexity: O(logn)
+#Space complexity: O(n)
+nums1 = [1,2,3]
+nums2 = [2]
+def intersect(nums1,nums2):
+    if (nums1 == None or nums2 == None or len(nums1) == 0 or len(nums2)==0):
+        return [0]
+    n1 = len(nums1)
+    n2 = len(nums2)
+    if n1 > n2:
+        return intersect(nums2,nums1)
+    sorted(nums1)
+    sorted(nums2)
```

Comment:
sorted returns the sorted array and does not modify the original, the code does not work.
Please use .sort() instead.